Framework/Algorithms/src/EstimateAsymmetryFromCounts.cpp

```diff
@@ -0,0 +1,157 @@
+//----------------------------------------------------------------------
+// Includes
+//----------------------------------------------------------------------
+#include "MantidAlgorithms/AsymmetryHelper.h"
+#include "MantidAlgorithms/EstimateAsymmetryFromCounts.h"
+#include "MantidAPI/IFunction.h"
+#include "MantidAPI/MatrixWorkspace.h"
+#include "MantidAPI/Run.h"
+
+#include "MantidAPI/Workspace_fwd.h"
+#include "MantidAPI/WorkspaceFactory.h"
+#include "MantidKernel/PhysicalConstants.h"
+#include "MantidKernel/ArrayProperty.h"
```

Comment:
do you mean to move ArrayProperty to be between WorkspaceFactorty and PhysicalConstants?